content/posts/hands-on-set-up-ingress-on-kubernetes-with-apache-apisix-ingress-controller.md

```diff
@@ -0,0 +1,327 @@
+---
+title: "Hands-On: Set Up Ingress on Kubernetes With Apache APISIX Ingress Controller"
+date: 2022-09-09T17:58:44+05:30
+draft: true
+weight: 14
+ShowToc: false
+mermaid: true
+summary: "A tutorial on using Ingress in your Kubernetes cluster with Apache APISIX."
+tags: ["ingress", "kubernetes", "apache apisix", "cloud-native"]
+categories: ["Featured", "API Gateway"]
+cover:
+  image: "/images/hands-on-set-up-ingress-on-kubernetes-with-apache-apisix-ingress-controller/gate-banner.jpeg"
+  alt: "Photo of a Japanese-style gate in the middle of a pond."
+  caption: "Photo by [Pixabay](https://www.pexels.com/photo/pagoda-in-gray-scale-shot-301614/)"
+  relative: false
+---
+
+In Kubernetes, [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) is a native object that allows you to access your services externally by defining a set of rules. Using a reverse proxy, an [Ingress controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) implements these defined rules and routes external traffic to your services.
+
+{{< mermaid >}}
+flowchart LR
+c(Clients) --> i("☸ Ingress")
+i --> s1("☸ Service 1")
+i --> s2("☸ Service 2")
+i --> s3("☸ Service 3")
+style i stroke: #e62129
+
+    subgraph k["☸ Kubernetes cluster"]
+    i
+    s1
+    s2
+    s3
+    end
+
+{{< /mermaid >}}
+
+[Apache APISIX](https://apisix.apache.org/) is an open source API gateway (a fancy reverse proxy) that provides features like authentication, traffic routing, load balancing, canary releases, monitoring, and more. APISIX also supports custom Plugins and integrates with popular open source projects like [Apache SkyWalking](https://apisix.apache.org/docs/apisix/next/plugins/skywalking/) and [Prometheus](https://apisix.apache.org/docs/apisix/next/plugins/prometheus/). To learn more about APISIX, you can see the [official documentation](https://apisix.apache.org/docs/apisix/getting-started/).
```

Comment:
Please either define an API Gateway or qualify the reverse-proxy as "souped-up", "improved", etc. instead of "fancy"